### ISSUE TITLE
hold_action not responding

### ISSUE THREAD

**Issue by martoq** [home-assistant/iOS#2103]
**iOS device model, version and app version**

Model Name: iPhone 11 Pro
Software Version: 15.3.1
App version: [2022.3]

**Home Assistant Core Version**

[2022.3.7]

**Describe the bug**
ios app does not respond to hold_action in lovelace.  Desktop works, ios app does not.

**To Reproduce**
hold button, no response

**Expected behavior**
hold button, action performed.

**Screenshots**

**Additional context**

This appears to have been an issue since January of last year based on comments in the forums.

**Comment by martoq** [comment]
Correction this appears to have been an issue since '20.

https://community.home-assistant.io/t/entity-button-card-hold-action-doesnt-work-on-ios/162337

**Comment by ZoltrixGFC** [comment]
Can confirm issue still exists.

**Comment by Nxt3** [comment]
This is still happening on more recent versions of HASSIO as well.

![image](https://user-images.githubusercontent.com/2940826/210139387-a50f2500-e8a5-43c2-b391-c289dba05f69.png)

**Comment by jperquin** [comment]
As of HA Companion App IOS version 2023.7 (2023.471) running on HA core 2023.10.0, I can confirm this issue is still present. Accessing same instance via Chrome-browser on IOS `hold_action` works as expected (just like on macos Chrome).

I really like to use the companion app for its additional funcionality, but having to think about which app / browser to use for a specific requirment is cumbersome to say the least..

**Comment by gregtakacs** [comment]
Still an issue in 2024.

**Comment by gregtakacs** [comment]
So my primary mobile is an Android and I only casually use iOS but now that I played with it a bit more the issue seems to be finger placement. With my iPhone if my finger moves even just a tiny smidge while holding, the blue highlight indicating "hold" just goes away and the hold action doesn't activate.

If I have the blue circle on the screen while releasing, the hold action works, but it's much harder to keep my finger steady than it is on Android.

The behavior is the same on both Chrome and in the companion app.

**Comment by bgoncal** [comment]
So my question was because if it also happens in the mobile browser this means we can't probably fix on the App side so you should report to frontend

**Comment by niobos** [comment]
I can confirm that I have the same behaviour in Mobile Safari and the iOS app. Holding down my finger on an entity doesn't bring up the expected `hold_action`; but it doesn't trigger the normal (short) tap action either. I just get a blue circle that disappears when I release.

I've tried for several dozen times to hold-without-moving, but wasn't able to trigger the hold_action even once. I also noticed that when I moved my finger purposely, the blue circle disappears. So this is a *different* behaviour from when I actually try to hold-without-moving. Based on the UI feedback, it does appear that I'm performing the hold-action (blue circle), but that it doesn't trigger the configured action.
